rfcs/0130-Implementing Checks API in tc-github while preserving Statuses API

```diff
@@ -0,0 +1,44 @@
+# RFC 130 - Implementing Checks API
```

Comment:
I think what happened here is that the RFC was an issue in #130 and a pr in #131.  @djmitche does the issue-to-pr system still work?

Reply:
I think Github finally broke that a few weeks ago.  I have a note to update the process in this repo.  But, since we have made a habit of naming after the PR, let's change this to 0131.

Reply:
what if instead of an issue, we open a placeholder PR for the whole discussion + writeup with a file called xxxx.md and then amend commit to get the right #.

Reply:
no worries!